src/test/java/org/amazon/library/PropertyReader.java

```diff
@@ -0,0 +1,17 @@
+package org.amazon.library;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertyReader {
+
+	public static String configReader(String Key) throws IOException {
```

Comment:
variable name should never begin with Uppercase, make it `key`

Reply:
Done.